decisions/adding-types.md

```diff
@@ -0,0 +1,15 @@
+# Adding types
+
+## Decision
+All components should have flow types _as well as_ `prop-types`. If you come across a file or component that isn't flow
+typed, submit a separate PR to add types to it (alongside any associated tests or `index.js` entry points). Don't
```

Comment:
I would add story files to the list as well

Reply:
good shout!